.github/workflows/LATEST_DEPENDENCY_VERSIONS.yml

```
@@ -6,28 +6,32 @@ on:
   workflow_dispatch:
 
 jobs:
-  test-without-yarn-lock:
-    name: Test without yarn.lock
+  test-without-pnpm-lock-yaml:
+    name: Test without pnpm-lock.yaml
     strategy:
       matrix:
         system: ["ubuntu-latest", "windows-latest"]
     runs-on: ${{ matrix.system }}
     steps:
+      - uses: actions/checkout@v2
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
       - uses: actions/setup-node@v2
         with:
           node-version: 16
-      - uses: actions/checkout@v2
-      - name: Delete yarn.lock
-        run: "rm yarn.lock"
+          cache: "pnpm"
+      - name: Delete pnpm-lock.yaml
+        run: "rm pnpm-lock.yaml"
       - name: Install
-        run: yarn
+        run: pnpm install
```

Just to be extra explicit, since we are doing it elsewhere:

```suggestion
        run: pnpm install --no-frozen-lockfile
```